automated_testing/report/emailAutomation.py

```diff
@@ -0,0 +1,245 @@
+import smtplib 
+import os
+from email.mime.multipart import MIMEMultipart 
+from email.mime.text import MIMEText 
+from email.mime.base import MIMEBase 
+from email import encoders 
+import base64
+import cv2
+
+# HTML based testing report
+def imageTag(pathToImage):
+    datauri = base64.b64encode(open(pathToImage, 'rb').read()).decode('utf-8')
+    imgtag = "<td width='37.5%'><img src='data:image/png;base64,{0}'></td>".format(datauri)
+    return imgtag
+
+def totalDevices(mainDir):
+    deviceCount = 0
+    file = open(os.path.join(mainDir,r'deviceId.txt'),'r')
+    for line in file:
+        deviceCount+=1
+    deviceCount-=2
+    return deviceCount
+    
+def totalTestCase(resultsDir,deviceId):
+    testCases = 0
+    testPassed = 0
+    resultFilePath = os.path.join(r"C:\Users\MAQUser\Desktop\scripts\start_execution\onebillion\results",deviceId,deviceId+'.txt')
+    file = open(resultFilePath,'r')
+    for line in file :
+        val =line.split('\n') 
+        if val[0] == "Pass" or val[0] == "Fail":
+            testCases+=1
+            if val[0] == "Pass":
+                testPassed+=1
+    return (testCases,testPassed)
+
+
+if __name__ == "__main__":
+    path = os.getcwd()
+    print(path)
+    mainDir = os.path.abspath(os.path.join(path, '..'))
+    print(mainDir)
+    totalTestCases = 94
+    fromaddr = "ayushm@maqsoftware.com"
+    toaddr = "ayushm@maqsoftware.com"
```

Comment:
Remove ayushm@maqsoftware.com and write Add from Email address here. Similarly do for To address